lib/x11regressiontest.pm

```diff
@@ -481,9 +481,20 @@ sub exit_firefox() {
 }
 
 sub start_gnome_settings {
-    send_key "super";
-    wait_still_screen;
-    type_string "settings";
+    my $is_sle_sp1 = check_var('VERSION', '12-SP1');
+    my $workaround_repetitions = 3;
+
+    do {
+        if ($is_sle_sp1) {
+            send_key "super";    # if launcher open, close (search string will also be removed).
+            send_key "esc";    # close launcher (after 'super' because pressing esc twice doesn't work as expected')
+        }
+        send_key "super";
+        wait_still_screen;
+        type_string "settings";
+        $workaround_repetitions--;
+    } while ($is_sle_sp1 && !check_screen 'settings' && $workaround_repetitions > 0);
```

Comment:
I think your validation run is stuck here in `testapi::check_screen(mustmatch='', timeout=30)` because you are missing parentheses. Also try to avoid check_screen without timeout. I suggest after the type_string add another `wait_still_screen(3)` and write in this line here `!check_screen('settings', 0) && $workaround_repetitions > 0`